README.md

```diff
@@ -272,7 +276,7 @@ For development, we use the [local version](http://docs.aws.amazon.com/amazondyn
 
 **config/dev.exs**
 
-```elixir               
+```elixir
 config :my_app, MyApp.Repo,
```

Comment:
@bernardd how does this jive with the logic in the definition of `ex_aws_config/1` in lib/ecto_adapters_dynamodb.ex (line 1647)? Specifically, this line -

```
config = Confex.get_env(:ecto_adapters_dynamodb, repo)
```

Am I correct that we need to change our documentation to indicate that the config that includes the `Repo` needs to point at `:ecto_adapters_dynamodb`, rather than `:my_app` (or whatever)?

Reply:
You know what? You are correct, but looking at this again I don't know what I was thinking. The standard ecto config does things this way, and I have no idea why I thought it was appropriate to change it. I'm gonna fix it so it's back the way it was (while still keeping the per-repo config, obviously).

Reply:
As discussed, you're right that this was entirely the wrong way to do it - I've fixed it up to be the way it was in The Before Times.